themes/navy/languages/fr.yml

```diff
@@ -0,0 +1,69 @@
+menu:
+  docs: Docs
+  api: API
+  news: Nouvelles
+  plugins: Plugins
+  themes: Thèmes
+  search: Chercher
+
+index:
+  get_started: Commençer
+
+page:
+  contents: Contenus
+  back_to_top: Retour au début
+  improve: Améliorer ce document
+  prev: Précédent
+  next: Suivant
+  last_updated: "Dernière mise à jour : %s"
+
+sidebar:
+  docs:
+    getting_started: Premier pas
+    overview: Vue d'ensemble
+    setup: Installation
+    configuration: Configuration
+    commands: Commandes
+    migration: Migration
+    basic_usage: Utilisation de base
+    writing: Écriture
+    front_matter: Matière Première
```

Comment:
I think this one should remain `Front Matter` as it is a technical term. Otherwise, `Métadonnées` sounds like a more appropriate translation to me.

Reply:
changed